real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/logging/OldLogConverter.java

```diff
@@ -0,0 +1,169 @@
+package com.roboclub.robobuggy.logging;
+import java.util.*;
+import java.io.*;
+
+
+public class OldLogConverter {
+	
+/** Steering Angle Conversion Rate */
+static private final int ARD_TO_DEG = 100;
+/** Steering Angle offset?? */
+static private final int OFFSET = -200;
+	
+static ArrayList<String> sensor_type = new ArrayList<String>();
+static ArrayList<Integer> sensor_quant = new ArrayList<Integer>();
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://www.codacy.com/app/Mr-Trevor-Decker/RoboBuggy/file/1331974238/issues/source?bid=2938665&fileBranchId=2957575#l14)